source/include/core_http_client.h

```diff
@@ -442,6 +451,21 @@ typedef struct HTTPResponse
      */
     HTTPClient_ResponseHeaderParsingCallback_t * pHeaderParsingCallback;
 
+    /**
+     * @brief Optional callback for getting the system time.
+     *
+     * This is used to calculate the elapsed time between network sends and
+     * receives greater than zero. If this field is set to NULL, then network
+     * send and receive won't be retries after a zero is returned.
+     *
+     * If this function is set, then the maximum elapsed time between network
+     * receives greater than zero is set in HTTP_RECV_RETRY_TIMEOUT_MS.
```

Comment:
```suggestion
     * If this function is provide, then the maximum time for retrying zero network 
     * reads can be set through the #HTTP_RECV_RETRY_TIMEOUT_MS configuration.
```

Reply:
Similar suggestion for the text on `HTTP_SEND_RETRY_TIMEOUT_MS`

Reply:
thanks